src/EditorFeatures/CSharpTest/AddFileBanner/AddFileBannerTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
what changed here?

Reply:
Unicode boms.  Happens sometimes when you do a find/replace.